api/views/workflow.py

```diff
@@ -32,11 +31,11 @@ def get_queryset(self):
         return [self.request.user, ]
 
 
-class ProviderViewSet(viewsets.ReadOnlyModelViewSet):
-    serializer_class = ProviderSerializer
+class SourceViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = SourceSerializer
 
     def get_queryset(self):
-        queryset = ShareUser.objects.exclude(robot='').exclude(long_title='')
+        queryset = Source.objects.exclude(user__robot='').exclude(long_title='')
```

Comment:
This excludes push sources, like before. But I now see there's a ticket to display push sources, so I could change this to `all()` and add yet another ticket to the title.